examples/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine
+EXPOSE 8080
+COPY hotrod /
+ENTRYPOINT ./main all
```

Comment:
This allows using only `all` command. Maybe it's better to use just `CMD` with `all`

Reply:
Fwiw I never tested HotRod as individual processes